js/map-content-to-topper.js

```diff
@@ -162,7 +162,7 @@ const hasDarkBackground = (backgroundColour) => {
 	return (darkBackgrounds.indexOf(backgroundColour) > -1);
 };
 
-module.exports = content => {
+export default content => {
```

Comment:
Are we waiting for a breaking to release these? Or should we add `export let content [...]` [like o-layers](https://github.com/Financial-Times/o-layers/pull/29)?

Reply:
Yes, we should export in both ways